deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -377,13 +377,17 @@ cephFileSystems:
         - failureDomain: host
           replicated:
             size: 3
+          # Optional and highly recommended, see https://github.com/rook/rook/blob/master/Documentation/ceph-filesystem-crd.md#pools
+          name: data0
       metadataServer:
         activeCount: 1
         activeStandby: true
     storageClass:
       enabled: true
       isDefault: false
       name: ceph-filesystem
+      # Specify a data pool to use, optional and highly recommended, must be the name of one of the data pools above
```

Comment:
I wonder if we should mention that the filesystem name will be automatically prepended, in case they look at the storage class? Perhaps it isn't necessary though since the chart will just keep it simple.